core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2013,6 +2013,40 @@ public String toString() {
           .setDescription("The maximum size of a message that can be sent to the Alluxio master")
           .setScope(Scope.MASTER)
           .build();
+    public static final PropertyKey MASTER_NETWORK_FLOWCONTROL_WINDOW =
+      new Builder(Name.MASTER_NETWORK_FLOWCONTROL_WINDOW)
+          .setDefaultValue("2MB")
```

Comment:
@jiacheliu3 This could potentially have an impact on your worker registration bechmarks.